pkg/filter/stream/dubbo/dubbo.go

```diff
@@ -0,0 +1,114 @@
+package dubbo
+
+import (
+	"context"
+
+	"mosn.io/api"
+	v2 "mosn.io/mosn/pkg/config/v2"
+	mosnctx "mosn.io/mosn/pkg/context"
+	"mosn.io/mosn/pkg/log"
+	"mosn.io/mosn/pkg/protocol"
+	"mosn.io/mosn/pkg/protocol/xprotocol/dubbo"
+	"mosn.io/mosn/pkg/types"
+	"mosn.io/pkg/buffer"
+)
+
+func init() {
+	api.RegisterStream(v2.DubboStream, buildStream)
+}
+
+type factory struct{}
+
+func buildStream(conf map[string]interface{}) (api.StreamFilterChainFactory, error) {
+	f := &factory{}
+
+	sskObj := conf[subsetKey]
+	if sskObj == nil {
+		return f, nil
+	}
+
+	ssk, ok := sskObj.(string)
+	if !ok {
+		return f, nil
+	}
+	podSubsetKey = ssk
+
+	return f, nil
+}
+
+func (f *factory) CreateFilterChain(ctx context.Context, callbacks api.StreamFilterChainFactoryCallbacks) {
+	filter := NewDubboFilter(ctx)
+	callbacks.AddStreamReceiverFilter(filter, api.BeforeRoute)
+	callbacks.AddStreamSenderFilter(filter)
+}
+
+type dubboFilter struct {
+	handler api.StreamReceiverFilterHandler
+}
+
+func NewDubboFilter(ctx context.Context) *dubboFilter {
+	return &dubboFilter{}
+}
+
+func (d *dubboFilter) OnDestroy() {}
+
+func (d *dubboFilter) OnReceive(ctx context.Context, headers api.HeaderMap, buf buffer.IoBuffer, trailers api.HeaderMap) api.StreamFilterStatus {
+
+	listener := mosnctx.Get(ctx, types.ContextKeyListenerName).(string)
+
+	service, ok := headers.Get(dubbo.ServiceNameHeader)
+	if !ok {
+		log.DefaultLogger.Errorf("This filter {%s} just for dubbo protocol, please check your config.", v2.DubboStream)
+		return api.StreamFilterStop
```

Comment:
if OnReceive returns StreamFilterStop , it means stop next stream filters, but do not stop the request.
Stop request should returns StreamFiltertermination. 
Just ingore the filter should  returns Continue, run the other filters.

Reply:
It should return StreamFiltertermination